code/src/Components/happyThought.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable no-underscore-dangle */
+import React, { useState, useEffect } from 'react';
+import moment from 'moment';
+import { THOUGHTS_URL } from '../App';
+import './happyThought.css'
+
+export const HappyThought = () => {
+  const [thoughts, setThoughts] = useState()
+
+  useEffect(() => {
+    fetch(THOUGHTS_URL)
+      .then((res) => res.json())
+      .then((data) => {
+        const filteredData = data.filter((thought) => thought.text !== '')
+        setThoughts(filteredData)
+      })
+  }, [])
+
+  const handleClick = (id) => {
+    fetch(`https://happy-thoughts-technigo.herokuapp.com/thoughts/${id}/like`, {
+      method: 'POST'
+    })
+      .then(() => {
+        window.location.reload()
+      })
+  }
+
+  return (
+    <div>
+      {thoughts && thoughts.map((thought) => (
+        <React.Fragment key={thought._id}>
```

Comment:
This is an interesting usage - React.Fragment, will look into it to learn more:)